### DISCUSSION TITLE
Ability to specify the recording directory per camera?

### DISCUSSION THREAD

**Discussion by phillipsnick** [blakeblackshear/frigate#6388] [Ask A Question]
I've been playing with the recording element of Frigate to see if I could replace using BlueIris for recording 24/7. 

Seems to tick all the boxes besides one, the ability to use multiple hard drives! Due to having fairly small drives and quite a few cameras it just made more sense to split this across multiple drives.

Is there a way to achieve this in Frigate? Or are there any plans for it?

If not I might have a play as looking around the code it seems this is what defines the location it gets stored? https://github.com/blakeblackshear/frigate/blob/dev/frigate/record/maintainer.py#L279

**Comment by NickM-27** [comment]
You could definitely use a fuse.fs or similar to combine multiple hard drives into one. But it's currently not supported to have different locations per camera.

**Comment by phillipsnick** [comment]
@NickM-27 I did look at something like this, or potentially mounting each camera as a different volume in docker. But due to the path structure I've not come up with anything that's going to work.

If it was possible to customise it so it's `/media/frigate/recordings/camera_name/YYYY-MM/` then it becomes fairly easy to solve this.

**Comment by ki9us** [comment]
I have a single 4TB HDD which I think is bottlenecking my recordings.  I'd like to switch to using three 2tb SDDs.  For six cameras, I could put two partitions per SSD and have more, faster storage.  

Using RAID, ZFS, LVM, FUSE, etc isn't viable because I want portability.  That is, I want to be able to unmount a single drive (with the camera I want), ang plug it into my PC to transfer recordings.  

This can probably be accomplished in one LoC, as phillipsnick suggests in the last post.  @NickM-27, is it ok to escalate this to a bug report?

**Reply by NickM-27** [reply]
I'm not aware of any bug here
